### PR TITLE
Feature/nmd 333 Remove comment in project

### PR DESCRIPTION
**Pull request by hinkevich**

### CODE REVIEW

File: GymApp/GYM.BLL.Tests/TestData/TestEntities.cs [hunk 1]
```diff
@@ -13,21 +13,21 @@ public static IEnumerable<CouchEntity> GetCouchEntitiesForTest()
                     Id = 1,
                     FirstName = "Arnold",
                     LastName = "Schwarzenegger",
-                    Describe = "Fitness, Bodybuilding",
+                    Description = "Fitness, Bodybuilding",
                     Visitors = new List<VisitorEntity>(),
                 },
                 new CouchEntity()
                 { Id = 2,
```
Comment (moltenessence) [inline GymApp/GYM.BLL.Tests/TestData/TestEntities.cs]: formatting

File: GymApp/GYM.BLL.Tests/TestData/TestEntities.cs [hunk 2]
```diff
@@ -13,21 +13,21 @@ public static IEnumerable<CouchEntity> GetCouchEntitiesForTest()
...
                     FirstName = "Mick",
                     LastName = "Smith",
-                    Describe = "Fitness, Cross-fit",
+                    Description = "Fitness, Cross-fit",
                     Visitors = new List<VisitorEntity>()
                 },
                 new CouchEntity()
                 { Id = 3,
```
Comment (moltenessence) [inline GymApp/GYM.BLL.Tests/TestData/TestEntities.cs]: formatting

File: GymApp/GYM.IntegrationTests/GymWebApplicationFactory.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                });
+                var sp = services.BuildServiceProvider();
+                using (var scope = sp.CreateScope())
+                using (var appContext = scope.ServiceProvider.GetRequiredService<GymAppDbContext>())
+                {
+                    try
+                    {
+                        appContext.Database.EnsureCreated();
+                    }
+                    catch (Exception ex)
```
Comment (moltenessence) [inline GymApp/GYM.IntegrationTests/GymWebApplicationFactory.cs]: variable ex is declared but never used

File: GymApp/GYM.IntegrationTests/IntegrationTests/CouchesControllerIntegrationTests.cs [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+            //Assert
+            response.StatusCode.ShouldBe(HttpStatusCode.OK);
+            responseString.ShouldContain(couch!.FirstName);
+        }
+
+        [Fact]
+        public async Task DeleteCouch_InputValidId_ReturnsNoContent()
+        {
+            //Arrange
+            await InitializeDb();
```
Comment (moltenessence) [inline GymApp/GYM.IntegrationTests/IntegrationTests/CouchesControllerIntegrationTests.cs]: you can avoid calling this method in each test case

File: GymApp/GYM.IntegrationTests/IntegrationTests/CouchesControllerIntegrationTests.cs [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+        private async Task InitializeDb()
+        {
+            IEnumerable<CouchEntity> couches = _fixture.Build<CouchEntity>().Without(p => p.Id).Without(p => p.Visitors).CreateMany(5).ToList();
+
+            _dbContext.CouchEntities.AddRange(couches);
+            await _dbContext.SaveChangesAsync();
+        }
+
+        //Get random couch
+        private CouchViewModel GetCouchViewModelForTest()
```
Comment (moltenessence) [inline GymApp/GYM.IntegrationTests/IntegrationTests/CouchesControllerIntegrationTests.cs]: try using autofixture as attribute with [Theory]

**Review by Genresal — APPROVED** [review]

**Review by moltenessence — APPROVED** [review]

**Review by stas1408123 — APPROVED** [review]

**Pull request merged**